src/main/resources/templates/index.html

```diff
@@ -42,8 +42,16 @@ <h1 class="govuk-heading-l" id="main-content" data-i18n="index:searchFilings">Se
                     <div class="govuk-form-group" th:classappend="${minFilingDateError != null} ? govuk-form-group--error : govuk-form-group--valid">
                         <fieldset class="govuk-fieldset" role="group" aria-describedby="min-filing-date-hint">
                             <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-                                <h1 class="govuk-fieldset__legend" data-i18n="index:dateFiled.header">
-                                    Date Filed
+                                <h1>
```

Comment:
arbitrary line:

The from and to elements are no longer aligned in your recording.